src/transport/MultiTransport.cpp

```diff
@@ -433,20 +434,20 @@ void MultiTransport::updateThermal_T()
     // evaluate polynomial fits for A*, B*, C*
     for (size_t i = 0; i < m_nsp; i++) {
         for (size_t j = i; j < m_nsp; j++) {
-            double z = m_logt - m_log_eps_k(i,j);
+            double z = m_star_poly_actualT[i][j] == 1? m_logt : m_logt - m_log_eps_k(i,j);
```

Comment:
Could you comment on what this modification does? (it's not mentioned in the PR description). Apart from a missing whitespace before the `?` I'm not sure what this does.

Further, commenting out the `m_om22` lines here without removing it from the header files leaves the possibility that it is used elsewhere. Specifically, the missing `resize` call could result in a segfault.

Reply:
If m_star_poly_actualT[i][j] is equal to 1 (or as you suggest below, if it is True), the polynomials are defined as a function of the logarithm of the actual temperature instead of the reduced temperature (T_ij = k_B*T/eps_ij).

Reply:
It seems like m_om22 is not used anywhere, so I removed it from the header files as well.